src/main/java/org/broadinstitute/hellbender/tools/AnnotatePairOrientation.java

```diff
@@ -0,0 +1,119 @@
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.engine.VariantWalker;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.tools.walkers.annotator.OxoGReadCounts;
+import org.broadinstitute.hellbender.tools.walkers.mutect.Mutect2Engine;
+import org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.genotyper.IndexedSampleList;
+import org.broadinstitute.hellbender.utils.genotyper.SampleList;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
+
+import java.io.File;
+import java.util.*;
+
+@CommandLineProgramProperties(
+        summary = "(Experimental) This will add fields normally provided by M2 to a VCF.  There should never be a need to run this tool on a VCF that was produced by M2." +
+                "\n  The output of this tool should be usable with FilterByOrientationBias." +
+                "\n  The output of this tool will only count reads that fully overlap (and match) the variant or reference sequence (this is relevant for indels)." +
+                "\n  IMPORTANT:  This tool cannot produce the exact same F1R2/F2R1 counts as you would get from M2, due to the nature of how M2 calls variants (using read likelihoods, wheras this tool uses a base quality filter).",
+        oneLineSummary = "(EXPERIMENTAL) Annotate a non-M2 VCF (using the associated tumor bam) with pair orientation fields (e.g. " + GATKVCFConstants.F1R2_KEY + " ).",
+        programGroup = VariantProgramGroup.class
+)
+@BetaFeature
+public class AnnotatePairOrientation extends VariantWalker {
+
+    @Argument(
+            doc = "Output Somatic SNP/Indel VCF file with additional annotations.",
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
+    protected File outputFile;
+
+    public final static String CUTOFF_SHORT_NAME = "c";
+    public final static String CUTOFF_LONG_NAME = "meanBaseQualityCutoff";
+    public final static double MEAN_BASE_QUALITY_DEFAULT_CUTOFF = 7.0;
+    @Argument(
+            doc = "Cutoff for the mean base quality value(s) to count the read.  These are for bases that overlap the variant.",
+            shortName = CUTOFF_SHORT_NAME, fullName = CUTOFF_LONG_NAME, minValue = 0.0, maxRecommendedValue = 10.0,
+            optional = true
+    )
+    private Double meanBaseQualityCutoff = MEAN_BASE_QUALITY_DEFAULT_CUTOFF;
```

Comment:
Why is this not a primitive `double`?

Reply:
oversight.  Reverted to `double`

Done